source/MapPlanetCard.h

```diff
@@ -86,10 +90,14 @@ class MapPlanetCard {
 	const Sprite *sprite;
 	float spriteScale;
 
+	std::string governmentName;
 	std::string reputationLabel;
 	const std::string &planetName;
 	// The currently select category (outfitter, shipyard, ...)
 	unsigned selectedCategory = 0;
+
+	static std::string systemGovernmentName;
+	static bool hasGovernments;
```

Comment:
Why are these static?
If they're going to stay static, they should probably just go in an anonymous namespace in MapPlanetCard.cpp.
But, I don't think they should be static at all.
`systemGovernmentName` doesn't even seem to need to be a class member at all. It is only ever set or accessed in the constructor, apart from the `ResetSize` static method, which just clears it, which wouldn't be something that needs to be done if it weren't a static class member but instead a local in the constructor.

Reply:
agreed, do you see how to move it to the namespace @TomGoodIdea ?

Reply:
It has to be a static because it must be the same for all planetcards: either they all disply it or not